pyproject.toml

```diff
@@ -44,6 +44,7 @@ nltk = "^3.5"
 stringcase = "^1.2"
 spacy = "^2.3.5"
 PyYAML = "^5.4.1"
+scikit-learn = "^0.24.2"
```

Comment:
An other big dependency, why not!

Reply:
Numpy doesn't have TruncatedSVD so our hand was forced. Although it's possible we could run full SVD with numpy at best it's a costly operation just to throw away the first item, and at worst it can cause a considerable performance lag. However, if we do adopt this embeddings approach, we might want to make the TfidfVectorizer available or wrapped so that users can easily and transparently create the idf mapping files. Doing Tfidf without sklearn is a non-starter ;-)